src/views/resources/list.html

```diff
@@ -132,4 +132,43 @@ <h6 class="text-danger">
       </div>
     </div>
   </div>
+
+  <div class="col-12 col-md-4">
+    <div class="card card-body" style="min-height: 118px">
+      <h6 class="text-info">
+        <span class="text-uppercase i18n">
+          <span data-lang="en_us">Subtotal on list</span>
```

Comment:
```suggestion
          <span data-lang="en_us">Subtotal</span>
```

Acho que não precisa ficar repetindo _na lista_ toda hora...

Reply:
subi agora pouco tirando